metadata-etl/src/main/resources/jython/HiveExtract.py

```diff
@@ -213,7 +223,8 @@ def format_table_metadata_v2(self, rows, schema):
                       TableInfo.native_name: row_value[22], TableInfo.logical_name: row_value[23],
                       TableInfo.dataset_name: row_value[18], TableInfo.version: str(row_value[19]),
                       TableInfo.serialization_format: row_value[2],
-                      TableInfo.create_time: row_value[3], TableInfo.db_id: row_value[4], TableInfo.table_id: row_value[5],
+                      TableInfo.create_time: row_value[3], TableInfo.source_modified_time: row_value[26],
```

Comment:
Since this dict is super long, let's have one assignment per line and align them to make it more readable.